sdk/containerregistry/azure-containerregistry/azure/containerregistry/_models.py

```diff
@@ -78,22 +79,24 @@ def _from_generated(cls, gen):
 class ArtifactManifestProperties(object):
     """Represents properties of a registry artifact
 
-    :ivar str cpu_architecture: CPU Architecture of an artifact
+    :ivar architecture: CPU Architecture of an artifact
+    :vartype architecture: ~azure.containerregistry.ArtifactArchitecture
     :ivar created_on: Time and date an artifact was created
-    :vartype created_on: :class:`~datetime.datetime`
+    :vartype created_on: datetime.datetime
```

Comment:
I think the `~` is required to get the link in documentation

```suggestion
    :vartype created_on: ~datetime.datetime
```